src/ui/src/app/Service/wolf-den.service.ts

```diff
@@ -100,4 +117,40 @@ export class WolfDenService {
       { headers: this.getHeaders(), params }
     );
   }
-}
+//notification
+  getNotification(employeeId: number): Observable<INotificationForm[]> {
+    const params = this.createHttpParams({
+      EmployeeId:employeeId
+    });
+    return this.http.get<INotificationForm[]>(
+      `${this.baseUrl}/api/Notification/employee`,
+      { headers: this.getHeaders(), params }
+    );
+  }
+  getTodaysAbsence():Observable<ItodaysAbsence[]>{
+    const params= this.createHttpParams({});
+    return this.http.get<ItodaysAbsence[]>(
+      `${this.baseUrl}/api/LeaveRequestDay/leaves-on-current-day`,
+      { headers: this.getHeaders(), params }
+    );
+  }
+  markAsRead(notificationId: number): Observable<any> {
+    const url = `${this.baseUrl}/api/Notification/read`;
+    const payload = { notificationId }; 
+    return this.http.patch<any>(url, payload, {
```

Comment:
why 2 extra constants can we do it directly

Reply:
done